src/lib/vocab/space/spaceServices.ts

```diff
@@ -21,6 +21,13 @@ import {
 	UpdateSpace,
 	DeleteSpace,
 } from '$lib/vocab/space/spaceEvents';
+import type {Community} from '../community/community';
+import type {Result} from '@feltcoop/felt';
+import type {Space} from './space';
+import type {ErrorResponse} from '$lib/util/error';
+import {toDefaultSpaces} from './defaultSpaces';
+import {db} from '$lib/db/db';
+import {type ISessionApi} from '$lib/server/SessionApi';
```

Comment:
```suggestion
import type {ISessionApi} from '$lib/server/SessionApi';
```

The SvelteKit upgrade requires we use this syntax when all of the imports to a module are types. (TypeScript will report an error)